composer.json

```diff
@@ -11,14 +13,15 @@
             "email": "matt@tighten.co"
         }
     ],
-    "license": "MIT",
     "require": {
-        "laravel/framework": "5.0.*|5.1.*|5.2.*|5.3.*|5.4.*|5.5.*|5.6.*"
+        "illuminate/config": "~5.6",
```

Comment:
Curious: What do we gain by dropping support for older versions of Laravel?

Reply:
Laravel 5.6 overhauled the [logging system](https://laravel.com/docs/5.6/logging), which broke the custom log file feature. I thought it would cleaner to make the switch instead of maintaining the older logging system. Thoughts?

Reply:
Makes sense. I think we may want to make it a major version update; would it be weird to switch to 2.0 four days after releasing 1.0?

Reply:
I suppose it would be a bit weird. However, even with `1.0` users on an older version of Laravel would hit an issue if they use custom log files. I'm not sure how to handle this. Should we make 1.0 require the use of Laravel 5.6 AND PHP7 and anything older has to use 0.*?

Reply:
We didn't have tests running against the custom log file feature. I wrote that in this PR.

Reply:
ahh... so if 1.0 will break for old laravel projects, we're good... other than that the 1.0 release will allow them to install, but then won't work. 😬 oh well.